seed/data_importer/tests/test_case_a.py

```diff
@@ -36,7 +38,12 @@ def setUp(self):
         self.fake_row = FAKE_ROW
         selfvars = self.set_up(import_file_source_type)
         self.user, self.org, self.import_file, self.import_record, self.cycle = selfvars
-        self.import_file.load_import_file(osp.join(osp.dirname(__file__), 'data', filename))
+        filepath = osp.join(osp.dirname(__file__), 'data', filename)
+        self.import_file.file = SimpleUploadedFile(
+            name=filename,
+            content=open(filepath, 'rb').read()
+        )
+        self.import_file.save()
```

Comment:
I assume this is due to the enhanced restrictions on resource data.  Seems good.